modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -241,23 +246,23 @@ exports.WebRequest = function (method, path) {
     this.method = WEB_UTILS.resolveMethod(dRequest.getMethod());
 
     this.path = function () {
-      var dRequestURI = dRequest.getRequestURI();
-      var filePlaceInURI = dRequestURI.search(XSJS_FILE_EXTENSION);
+      const dRequestURI = dRequest.getRequestURI();
+      const filePlaceInURI = dRequestURI.search(XSJS_FILE_EXTENSION);
       return dRequestURI.substring(0, filePlaceInURI + XSJS_FILE_EXTENSION_LENGTH + dRequestURI.length);
     }();
 
     this.queryPath = function () {
-      var dRequestURI = dRequest.getRequestURI();
-      var filePlaceInURI = dRequestURI.search(XSJS_FILE_EXTENSION);
+      const dRequestURI = dRequest.getRequestURI();
+      const filePlaceInURI = dRequestURI.search(XSJS_FILE_EXTENSION);
       return dRequestURI.substring(filePlaceInURI + XSJS_FILE_EXTENSION_LENGTH, dRequestURI.length);
     }();
 
     this.headers = function () {
-      var dHeadersArray = [];
-      var headerNamesArray = dRequest.getHeaderNames();
+      let dHeadersArray = [];
```

Comment:
Use `const`

Reply:
Values are pushed to dHeadersArray

Reply:
This is not correct. const means the variable can't be reassigned, not that you can't modify it. Refer to MDN for additional docs